README.md

```diff
@@ -1,4 +1,6 @@
 [![Build Status](https://ci.centos.org/buildStatus/icon?job=devtools-rh-che-build-che-credentials-master)](https://ci.centos.org/view/Devtools/job/devtools-rh-che-build-che-credentials-master/)
```

Comment:
What about having any labels (displayed by default or by hoovering)? It is not clear which job is failing seeing only:

![screenshot from 2018-06-28 13-00-48](https://user-images.githubusercontent.com/29278800/42030521-5986e1e2-7ad3-11e8-9691-e32787895749.png)

Reply:
MD format does not support hovering afaik (it is possible to add it by inlining html but this is not recommended )